packages/Popover/README.md

```diff
@@ -50,10 +50,11 @@ npm install @paprika/popover
 
 ### Popover.Content
 
-| Prop     | Type | required | default  | Description                                  |
-| -------- | ---- | -------- | -------- | -------------------------------------------- |
-| children | node | false    | null     |                                              |
-| onBlur   | func | false    | () => {} | Callback to indicate the element loses focus |
+| Prop      | Type | required | default  | Description                                  |
+| --------- | ---- | -------- | -------- | -------------------------------------------- |
+| children  | node | false    | null     |                                              |
+| onBlur    | func | false    | () => {} | Callback to indicate the element loses focus |
+| onKeyDown | func | false    | () => {} |                                              |
```

Comment:
Let's not expose this one, don't declare it on the props, this gives the wrong impression about where to put the onKeyDown when composing the Popover. 

Normally when someone wants to use the onKeyDown and onKeyUp want to do it over the `<Popover.Content />` which gets the focus after the popover is open not to the `<Popover>` itself. 

also, all of our components should be able to spread this onClick, onKeyDown etc. So should be obvious that we can pass down those type of Props. 

`<sc.Popover data-pka-anchor="popover" {...moreProps} ref={this.$popover} onKeyDown={this.handleKeyDown}>`

Reply:
Ok will just call `moreProps.onKeyDown` when needed to avoid declaring it on props